src/hb-ot-layout-gsubgpos.hh

```diff
@@ -2002,6 +2002,77 @@ struct ChainRule
 					      lookup.arrayZ, lookup_context));
   }
 
+  void serialize (hb_serialize_context_t *c,
```

Comment:
This isn't really a serialize method since it's being called on the source object, recommend renaming to copy() or copy_and_remap()